lib/modules/keyboardNav.js

```diff
@@ -1219,7 +1219,13 @@ function reply(selected = getSelected()) {
 	));
 }
 
-function navigateTo(newWindow, href) {
+function navigateTo(newWindow, href, leaveNP = false) {
+	if (leaveNP) {
```

Comment:
Could you also verify that the NoParticipation module and its `escapeNP` option are both enabled? There should be a helper like "get opaque option value", there are some other modules which check options from other modules. (Let me know if you'd like a pointer.)

Reply:
Even better, could you add a new `escapeNPKeyboardNav` option right after it, and add a new human-readable title/description to `en.json`?  You can just copy paste and add a qualifier like "when using keyboard navigation to go to another page."

And for the coup de gras: could you also add a new "data migration" to copy the current value of the `escapeNP` option into the new option. `migrate.js` is full of examples.

Reply:
If we are introducing a new escapeNPKeyboardNav option:

Some users might be confused that the escapeNP option has no effect on keyboard nav. Should I edit its description to say it only affects link clicks?

Reply:
> Could you also verify that the NoParticipation module and its `escapeNP` option are both enabled? There should be a helper like "get opaque option value", there are some other modules which check options from other modules. (Let me know if you'd like a pointer.)

Added this in commit #4789f18

Reply:
Hmm, they might be confused. Updating a description is tricky, though, since the other translations won't mention the link clicks. I think it's an acceptable compromise to just add the new option and leave the old one alone.

It would probably help to make `escapeNPKeyboardNav` depend on `escapeNP` being enabled. There's a pattern like `enabled => module.options.escapeNP.value`.

Reply:
Let's not bother with a new option for this PR.